web_01.html

```diff
@@ -1,68 +1,66 @@
+                                    <ul>
+                                        <li><a href="#">TEMPLATE FEATURES</a></li>
+                                        <li><a href="#">TESTIMONIALS</a></li>
+                                        <li><a href="#">OUT STAFF</a></li>
+                                        <li><a href="#">404 NOT FOUND</a></li>
+                                    </ul>
+                                </li>
+                                <li><a href="#">CONTACT</a></li>
+                                <li><a href="#"><i class="fa-solid fa-bag-shopping"></i> $180</a>
+                                    <ul>
+                                        <li><a href="#"><img src="/images/bag.jpg" width="40px"></img>2 x $20 = $40</a></li>
+                                        <li><a href="#"><img src="/images/woman.jpg" width="40px"></img>2 x $20 = $40</a></li>
+                                    </ul>
+                                </li>
+                            </ul>
+                        </nav>
                     </div>
                 </div>
-            </div>
         </header>
         <main>
             <section class="main__top">
-                    <div class="arrow__icons">
-                        <a href="#"><i class="fa-solid fa-angle-left"></i></a>
-                        <a href=""><i class="fa-solid fa-angle-right"></i></a>
-                    </div>
-                    <div class="socmed__icons">
-                        <ul>
-                            <li><a href="https://facebook.com/" target="_blank"><i class="fa-brands fa-facebook-f"></i></a></li>
-                            <li><a href="https://twitter.com/" target="_blank"><i class="fa-brands fa-twitter"></i></a></li>
-                            <li><a href="https://google.com/" target="_blank"><i class="fa-brands fa-google-plus-g"></i></a></li>
-                        </ul>
-                    </div>
-                    <div class="main__greetings">
-                        <a href="#"><h1>Hello, my minimal friend!</h1></a>
-                    </div>  
+                <div class="arrow__icons">
+                    <a href="#"><i class="fa-solid fa-angle-left"></i></a>
+                    <a href=""><i class="fa-solid fa-angle-right"></i></a>
+                </div>
+                <div class="socmed__icons">
+                    <ul>
+                        <li><a href="https://facebook.com/" target="_blank"><i class="fa-brands fa-facebook-f"></i></a></li>
+                        <li><a href="https://twitter.com/" target="_blank"><i class="fa-brands fa-twitter"></i></a></li>
+                        <li><a href="https://google.com/" target="_blank"><i class="fa-brands fa-google-plus-g"></i></a></li>
+                    </ul>
+                </div>
+                <div class="main__greetings">
+                    <a href="#"><h1>Hello, my minimal friend!</h1></a>
+                </div>  
```

Comment:
Incorrect use of BEM methodology

```suggestion
                <div class="main__arrow-icons">
                    <a href="#"><i class="fa-solid fa-angle-left"></i></a>
                    <a href=""><i class="fa-solid fa-angle-right"></i></a>
                </div>
                <div class="main__socmed-icons">
                    <ul>
                        <li><a href="https://facebook.com/" target="_blank"><i class="fa-brands fa-facebook-f"></i></a></li>
                        <li><a href="https://twitter.com/" target="_blank"><i class="fa-brands fa-twitter"></i></a></li>
                        <li><a href="https://google.com/" target="_blank"><i class="fa-brands fa-google-plus-g"></i></a></li>
                    </ul>
                </div>
                <div class="main__greetings">
                    <a href="#"><h1>Hello, my minimal friend!</h1></a>
                </div>  
```